resource_management_mobile/src/app/pages/resource-hiring/view-hiring/view-hiring.component.scss

```diff
@@ -0,0 +1,143 @@
+@import "../../../../theme/theme.scss";
+
+.shadow {
+    -webkit-box-shadow: 4px 6px 13px 5px gray gray;
+    -moz-box-shadow: 4px 6px 13px 5px gray gray;
+    box-shadow: 4px 6px 13px 5px gray gray;
+}
+
+.card-user-info {
+    display: flex;
+    flex-direction: column;
+    line-height: 22px;
+    padding: 11px;
+}
+
+.user{
```

Comment:
If the class is repeating then create a common style and inject it in angular.json

Reply:
Done